docs/src/main/sphinx/release/release-414.md

```diff
@@ -20,10 +20,11 @@
 
 ## Security
 
-* Disallow writing system information with the `default` system access control.
-  System information writes can be re-enabled by setting the
-  `access-control.name` configuration property to `allow-all`, or by configuring
-  [system information rules](system-file-auth-system_information). ({issue}`17105`)
+* Disallow requesting [graceful shutdown](/admin/graceful-shutdown) with the
+  `default` [system access control](/security/built-in-system-access-control).
+  Shutdowns can be re-enabled by using the `allow-all` system access control, or
+  by configuring [system information rules](system-file-auth-system_information)
+  with the `file` system access control. ({issue}`17105`)
```

Comment:
I think we need to stress the fact it has to be configured on worker nodes. People may expect that configuring access control on coordinator is enough.

Or do you think docs/src/main/sphinx/security/built-in-system-access-control.rst changes are enough?

Reply:
I think ultimately the release notes should provide enough links to go and read more and learn the exact steps necessary from elsewhere in the docs.